README.md

```diff
@@ -26,6 +26,18 @@ const apos = require('apostrophe')({
 });
 ```
 
+#### Setting the `baseUrl`
+
+It is important to [set the `baseUrl` option](https://docs.apostrophecms.org/reference/core-server.html#baseurl) on your ApostropheCMS application for various reasons. In the SEO module it contributes to building the correct `canonical` link tag URL. This can be set on the main app configuration in `app.js`, however it is usually more appropriate to [set it in the `data/local.js` file](https://docs.apostrophecms.org/core-concepts/global-settings/settings.html#changing-the-value-for-a-specific-server-only) as that will contain environment/server-specific configurations.
```

Comment:
Alternatively you might consult an environment variable in `app.js`.

Reply:
Is this different from what I suggested with "This can be set on the main app configuration in `app.js`, however..."?

Reply:
It is different because if you use an environment variable which is set differently in production to determine the setting, it is not "more appropriate" to use data/local.js. They are equal approaches, and the env var is more common in cloud settings.

Reply:
Got it. I'm not going to go into too much detail with every variation since this isn't the best place for full-on `baseUrl` documentation.